### PR TITLE
[Bulky Goods] Add link to Household Recycling Centre to bin days page

### PR DESCRIPTION
**Pull request by nephila-nacrea**
Closes https://github.com/mysociety/societyworks/issues/3311.

[skip changelog]

### CODE REVIEW

File: templates/web/peterborough/waste/_more_services_sidebar.html [hunk 1]
```diff
@@ -38,6 +38,11 @@ <h3>More services</h3>
     <li>
         <a href="[% c.uri_for_action('waste/bulky', [ property.id ]) %]">Book bulky goods collection</a>
     </li>
+    [% IF c.cobrand.moniker == 'peterborough' %]
```
Comment (davea) [inline templates/web/peterborough/waste/_more_services_sidebar.html:41]: This is in the `peterborough` directory so doesn't need the cobrand check :)

**Review by davea — CHANGES_REQUESTED** [review]
One little `IF` to be removed is all!

File: templates/web/peterborough/waste/_more_services_sidebar.html (hunk 1 above)
Comment (nephila-nacrea) [reply templates/web/peterborough/waste/_more_services_sidebar.html:41]: 🤦 Don't know how I didn't notice that.

Comment (nephila-nacrea) [reply templates/web/peterborough/waste/_more_services_sidebar.html:41]: The ticket didn't actually state where the link ought to go so I assume it is OK where it is.

Comment (davea) [reply templates/web/peterborough/waste/_more_services_sidebar.html:41]: Yeah I think that's the perfect place to put it.

**Review by davea — APPROVED** [review]

**Pull request merged**